setup.py

```diff
@@ -157,30 +160,30 @@ def write_version(filename=os.path.join(*['airflow',
 datadog = ['datadog>=0.14.0']
 doc = [
     'mock',
-    'sphinx>=1.2.3',
     'sphinx-argparse>=0.1.13',
+    'Sphinx-PyPI-upload>=0.2.1',
     'sphinx-rtd-theme>=0.1.6',
+    'sphinx>=1.2.3',
     'sphinxcontrib-httpdomain>=1.7.0',
-    'Sphinx-PyPI-upload>=0.2.1'
 ]
 docker = ['docker~=3.0']
 druid = ['pydruid>=0.4.1']
 elasticsearch = [
+    'elasticsearch-dsl>=5.0.0,<6.0.0',
     'elasticsearch>=5.0.0,<6.0.0',
-    'elasticsearch-dsl>=5.0.0,<6.0.0'
 ]
 emr = ['boto3>=1.0.0, <1.8.0']
```

Comment:
Is this referred to anywhere? The extra for this was removed.

Reply:
I forgot about this line.